src/react-imgix.js

```diff
@@ -38,6 +38,7 @@ const COMMON_PROP_TYPES = {
 
 const SHARED_IMGIX_AND_SOURCE_PROP_TYPES = {
   ...COMMON_PROP_TYPES,
+  aspectRatio: PropTypes.number,
```

Comment:
It seems like `aspectRatio` is a string

Reply:
This was fixed in a commit I didn't push 😬